### PR TITLE
Remove Sarah Weinstein

### PR DESCRIPTION
**Pull request by MerricdeLauney**
Remove Sarah Weinstein per her request

### CODE REVIEW

**Comment by MerricdeLauney** [comment]
Will become a PR once I have visible approval from @sweinstein22

**Comment by sweinstein22** [comment]
yes please, thanks!

**Review by Gerg — APPROVED** [review]

**Pull request merged**
